### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by Lioness100**
I fixed one or two typos 😉 (this was a trial run for [a CLI I made](https://github.com/Lioness100/rspell))
Uh.. sorry for making you review all this loool

### CODE REVIEW

File: automod/conditions.go [hunk 1]
```diff
@@ -318,7 +318,7 @@ func (cd *ChannelCategoriesCondition) MergeDuplicates(data []interface{}) interf
 /////////////////////////////////////////////////////////////////
 
 type AccountAgeConditionData struct {
-	Treshold int
+	Threshold int
```
Comment (jo3-l) [inline automod/conditions.go:321]: We cannot rename effect/trigger data fields; they are serialized and saved to database so existing data would no longer deserialize correctly after this change. Please revert.

File: lib/cardsagainstdiscord/deck_football.go [hunk 2]
```diff
@@ -15,7 +15,7 @@ func init() {
 			&PromptCard{Prompt: `I love %s so much I would lend him my children.`},
 			&PromptCard{Prompt: `Come place your bets, ladies and gentlemen! %s vs. %s. Who will win?`},
 			&PromptCard{Prompt: `What's worse then being a Spurs fan? %s.`},
-			&PromptCard{Prompt: `Rumors say if you say %s infront of a mirror 3x your club will win a quadruple.`},
+			&PromptCard{Prompt: `Rumors say if you say %s in fronr of a mirror 3x your club will win a quadruple.`},
```
Comment (jo3-l) [inline lib/cardsagainstdiscord/deck_football.go:18]: Neither the original nor the proposed text is correct.
```suggestion
			&PromptCard{Prompt: `Rumors say if you say %s in front of a mirror 3x your club will win a quadruple.`},
```

File: lib/go-twitter/README.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ tweets, resp, err := client.Timelines.HomeTimeline(&twitter.HomeTimelineParams{
 })
 
 // Send a Tweet
-tweet, resp, err := client.Statuses.Update("just setting up my twttr", nil)
```
Comment (jo3-l) [inline lib/go-twitter/README.md:56]: This is intentional; the original tweet referenced here has this spelling. See <https://twitter.com/jack/status/20>.

**Review by jo3-l — CHANGES_REQUESTED** [review]

**Comment by lemm-e** [comment]
Sending a PR like this, with changes in actual code and comments all at once, while testing and clearly promoting your personal project here, doesn't sit very well.

**Comment by ashishjh-bst** [comment]
too big of a PR to do in a single change, cannot be tested in one go, raise smaller moduler PRs.

**Comment by Lioness100** [comment]
How modular do you want?

**Pull request closed without merging**